### DISCUSSION TITLE
Groups.io has a 100-person limit - should we resolve this and if so, how?

### DISCUSSION THREAD

**Discussion by w7sst** [w7sst/MorseRunner#369] [General]
We opened the groups.io discussion page in October of 2022 and we reached the 100-person limit only 15 months later in January of 2024. When new users try to join the group, they get an error message saying the group is full.

We are currently using the groups.io free plan. The next step is their Premium Plan at $220 US per year. You can find their pricing page [here](https://groups.io/static/pricing).

Some of my initial ideas include:
- Enable the feature allowing group members to [sponsor the Groups.io hosting fees](https://groups.io/helpcenter/ownersmanual/1/using-the-sponsorship-feature-to-defray-groups).
- Enquire if any clubs would like to sponsor this group.
- Ask the current groups.io membership if they feel this group is adding value to MRCE.
- Ask the current groups.io membership if they would be willing to sponsor the group each year at a rate of $2-3/yr.
- Partner with the various training groups (e.g. CWOPS) using MRCE as a teaching tool to see if they could help us by sponsoring us. We can continue adding features/contests that help their training and teaching.
- Ask ourselves the hard question if this group is valuable long-term?

I'd like to get your feedback and ideas on what we should do about this.

Thank you,
Mike W7SST

**Comment by w7sst** [comment]
### Contents of an email discussion with Scott in March 2024:

Hi Scott,

Have you noticed that we are no longer getting new members joining the public group?

I noticed that we now have 99 members in [groups.io/MorseRunnerCE](http://groups.io/MorseRunnerCE). We are currently using the Free Plan which allows upto 100 members. We can upgrade to the Premium Plan which allows up to 500 members for $220/yr (billed annually). Details can be seen here: https://groups.io/g/MorseRunnerCE/plans

We can enable Group Sponsorship which allows group members to help sponsor the cost of the Premium membership.

Or, should we consider backing off on the [groups.io](http://groups.io/) site. I think [groups.io](http://groups.io/) membership will continue to grow as additional releases continue to be made. We had good momentum going a year ago and it slowed down with my coding absence.

I was wondering if you have any suggestions on how to proceed. What we need is a sponsor and a few sponsors to help pay this cost.

Do you have time to help draft a letter/message that we can post both on [groups.io](http://groups.io/) and on [github.com](http://github.com/) asking for help paying this cost? Or an email to the core group, Jim, Bob, Laurent?

Any thoughts or comments.

Thank you,
Mike


### Scott's reply...
Mike, 

I had no idea Groups.io had a membership limit. $220 a year seems a bit much. I'm at a loss on an alternative. Maybe if enough people complain about the 100 member limit they will change it. 

It does seem redundant to github's discussions section. It also didn't work as easily because in the discussions area I can take a discussion and turn it into an issue. But in [groups.io](http://groups.io/) I have to copy and paste into github. That and all our downloads are located on github so in order for anyone to get our program they have to go to https://github.com/w7sst/MorseRunner/releases anyway. 

73
KD4SIR
Scott Hibbs

**Comment by w7sst** [comment]
### Feedback and question from Frank, @W4Tg, Sept 28, 2024

> Mike,
> 
> you didn't respond to my question about why the groups.io was limited to
> 100 people.  Was that by intent? Happy to help chip in, if that was the
> reason.
> 
> Regards,
> Frank

### Mike's response dated Sept 28, 2024...
Thank you for reminding me...

Turns out Groups.io has a 100-person limit for their free account. We would
have to upgrade to their Premium Account at a cost of $200/year. We will
start a Discussion Page on GitHub to brainstorm some ideas and get a plan
together. Groups.io allows us to put a Sponsor-this-group link on the
group's home page and allow members to contribute. I'm also wondering if we
could find some clubs that might be interested in sponsoring us. We also
have some groups using MRCE for CW Training courses, perhaps they could
donate as well. Of course, we have to be sure there is sufficient value in
this groups.io page to justify people's donations.

If you have any ideas, let me know.

We are also looking for someone to head up the discussion.

Thank you and 73,
Mike W7SST

### Frank's reply...
> Also cwops might help as they use it in their training.   I’ll talk to a few friends and get their thoughts.

**Comment by W4Tg** [comment]
Mike, I checked with K3CT of the N1MM Plus project and he said " We periodically post a message asking for donations. The money that we collect in a few hours is usually enough for more than a year of expenses."   
Could we do something similar?  I'm certainly willing to donate, but not being part of the current MorseRunner groups.io, I can't suggest this to the existing group.  Do you track the emails of all the people that download the releases?  Is there a way to write to them?  I guess the question is also do you have a way to collect and manage the money.  And deal with any tax issues. 

73,
Frank W4TG

**Comment by scotthibbs** [comment]
My two cents: 

I don't see any value added by having a groups.io account... Especially since this Github Discussions Area already provides all that groups.io offers. Asking a person to join Groups.io vs Github is no different except Github is a little more secure. We have to direct everyone to Github to download the latest release already, so all of our users already know about this site. Besides, it's quite confusing having both places to provide discussions and we can't create an issue from a discussion in Groups.io. 

So... I think the real question is do we want a webpage separate from Github? Again I would say no, apart from a hi-score website with links to Github for all other needs. 

Scott
KD4SIR

**Reply by W4Tg** [reply]
Either provides the functionality.  However, I don't know of other ham sw using github for discussions with users.   N1MM, WSJTX, eznec, and all my other ham sw uses groups.io now.  Perhaps because some, such as the Elecraft sw, are not open source projects.   If you want to use github, then the MRCE app help menu should have a pointer to the github discussion page, since it is not visible from the readme page.  I contacted Mike via email since I didn't know to find the github discussion page.

**Comment by w7sst** [comment]
I would like to stick with groups.io for our user-group discussion area. I am a member of many groups on groups.io, most of them are ham related. I appreciate getting the emails when a topic starts or continues. I agree that I have not seen many projects using GitHub for non-developer users.

It seems like we should perhaps post a Fundraiser topic and ask for contributions from the user group. We should tell them we have hit the 100-person limit and that we are trying to raise $220/year to cover the hosting costs. We can also ask the group if they feel this community is worth funding and keeping on groups.io. If there is not interest, then it will have to fall back to GitHub.

Second option is to post a discussion topic on groups.io and ask if members are willing to support the hosting fees through contributions. BTW, the fundraising support is built into groups.io and they handle it for us. Any money collected goes towards our hosting costs.

Can someone help us draft a few paragraphs that I can post on groups.io? 

Thank you,
Mike

Adding @K6OK @f6fvy @jr8ppg @WR7Q

**Comment by W4Tg** [comment]
Any idea how many downloads a typical release gets? That is, do we think
the 100 current groups.io represents 20% of the MRCE users?

Because as a non-member of the mrce groups.io I won’t see the funding
request, I wonder if it is possible to open up the group immediately and
then post the funding request?  I can cover the $20 for the first month for
the 500 member level.

73,
Frank
W4Tg

**Comment by K6TD** [comment]
I’ll take the 2nd month, and probably a few more…

I get plenty of groups.io email, as well as plenty of GitHub email.

73, K6TD

**Comment by w7sst** [comment]
We can get the first 90 days of the Premium Plan for free. This will give us plenty of time for the fundraiser to raise the hosting funds.

**Comment by w7sst** [comment]
Several of you have asked about traffic. The 2-week MRCE GitHub traffic is available [here](https://github.com/w7sst/MorseRunner/graphs/traffic). The two screenshots below (which are taken from this page) show the traffic, top referring sites and popular content over the past two weeks.

![Screenshot 2024-10-09 223959](https://github.com/user-attachments/assets/d9cd8fa3-f157-477e-83f2-9b866f825613)

![Screenshot 2024-10-09 224034](https://github.com/user-attachments/assets/5b6deaf1-9c76-460c-ace6-3495c128a098)

MRCE 1.85 was released on Sept 28, so you can see the traffic increase immediately after the release. There was visitor activity prior to the release. I think that was from our checkin activity leading up to the release date. I have often seen 2-4 clones per week occurring over time.

#### Observations
- There has been 465 unique visitors to the v1.85 release page.
- The top referring site contains a [posting announcing the MRCE 1.85 release](https://www.hamlife.jp/2024/10/01/morse-runner-ja-185/) (Thank you to whomever posted this). This site has brought in nearly 300 unique visitors. Be sure to click the English translation.
- Next in line with 46 unique visitors is groups.io (currently 100 members). This would not include counts of people that searched the internet and happened to link to a page within groups.io (i.e. they may not be a current MRCE member).
- Github.com has provided 11 unique visitors. I am not sure how many people are following the github group, nor how many are finding this from general google searches.
- Today's ARRL Contest Update has included a mention of the 1.85 release.

Back to our current topic of groups.io, the data above suggests that 10% of the visitors to the 1.85 release page perhaps came from groups.io. Remember that the group filled up at 100 users in February. If the group had continued to grow, we would have even more visitors from groups.io.

**Reply by jr8ppg** [reply]
Hi
I provided the information.
hamlife.jp is an information site about amateur radio in Japan.
Morse Runner is a popular CW practice software in Japan.
Thank you for all your improvements.

**Comment by K6OK** [comment]
As an alternative to groups.io, how about creating a forum on Reddit?  It's free with no traffic limit.  One of the options that a user can enable is to receive daily digests by e-mail.  Another very useful option that the subreddit owner can enable is to host a wiki (MRCE really could use a wiki for new users).   /r/Morserunner is available.  To get a feel for how it looks:

Reddit compact interface: https://www.reddit.com/r/amateurradio/?feedViewType=compactView
"Old" style interface (I prefer this one): https://old.reddit.com/r/amateurradio/
wiki: https://www.reddit.com/r/amateurradio/wiki/index/

If MRCE has a news announcement (like a new release), it's easy to cross-post it to /r/amateurradio, which has 150,000 subscribers.

Just my 2 cents....

**Comment by w7sst** [comment]
Well that's an interesting idea to utilize a Reddit forum. I personally have not looked at nor used Reddit, however, I have visited google links into the various Reddit forums. I like the 150k users - this helps solve our problem of how do we announce a new release to the most people. Other than our efforts to post to smaller newgroups/clubs/emails, etc., we don't have a good way to announce each release.

I also like the idea of a wiki, but don't have the time to build it. I guess that is why we call this a Community Edition.

Let's keep the ideas coming for a while longer.

Should we ask a question on groups.io and see if the existing groups.io members have any input or feedback regarding staying on groups.io or moving the user-group to Reddit?

**Comment by K6OK** [comment]
Mike, et al -- I'm not suggesting Reddit is a better choice than groups.io, only that it's a no-cost alternative.  Honestly with only 19 posts in 2024 I'm not sure $220/year is worth it, unless the team can find a sponsor.  Otherwise, the team will have to do the annual begging posts, like you see on some groups.io forums.

@w7sst @scotthibbs

**Reply by W4Tg** [reply]
I am on Reddit a bit but it doesn't feel the same to me as groups.io for community functions.  The separate issue Mike and K6OK raised is more of a marketing one.   I believe CWA advisors all know about MRCE.  My local club (VWS) which runs cw training also mentions it.  Given the latest release, perhaps a reminder to the ARRL affiliated clubs as well as to CWA might be in order.  I'd rather have the new forum (Groups.io or whatever) up before initiating that though, so we have some place to point the people to.  
- frank W4Tg
- 
@w7sst @K6OK

**Comment by w7sst** [comment]
I think I would like to see us proceed with groups.io. I'd like to give this a chance and see if user's would support the hosting fees. If not, we can fall back to reddit.

To get this started...
- I have enabled the Group Sponsorship button for our group. I have not seen any donations as of yet.
- I'd like to switch over to the Premium plan and utilize their 90 day free hosting days.
- I'll submit a group message asking for people to sponsor the group hosting fees. 
- If we see sufficient interest and support, we can go forward with this. If there are no sponsors, I'll pull my credit card before it is billed.

Also
- We should probably reserve reddit's *MorseRunnerCE* or *MorseRunner* names. Having these as a backup would be a good plan.
- I agree with Frank, W4Tg, that we should have a landing page before further announcements. 
- I have a 1.85 bug fix that I'd like to release as 1.85.1 at the end of this month. I'll wait on posting 1.85.1 until we have a solid plan for the user's group.
- In the 1.85.1 release notes, we can announce removal the groups.io 100-person limit and ask users to consider sponsoring the user's group.
- I also like the idea the idea of announcing MRCE on reddit's /r/amateurradio page. 
- Also like the idea of announcing to the ARRL Affiliated clubs (However, I admit I do not know how to do this).

Please let me know what you think of this plan.

Thank you,
73 Mike W7SST

@scotthibbs @k6ok @W4Tg @K6TD

**Comment by W4Tg** [comment]
Sounds good.  I’ll check with the Affiliated Club person for VA since he
belongs to my local club.

73,
Frank W4Tg

**Comment by W4Tg** [comment]
Here is the response:

There is two ways of reaching out to the ARRL Affiliated Club Coordinators
...



   - Reach out to Mike Walters at the League and ask him to push this out
   to the ACCs (he has a ARRL group mailing list)
   - Write a short 2-3 para short blurb for the ARRL Club mailing list



Either way, I can help expedite w/in the League



Mike, WA8AHZ

ARRL VA Section Affiliated Club Coordinator (NOVA)



OK to pass my email to Mike Brashler ....

Once we have groups.io up we can ask the League and RAC and others to
announce it.  Also the contest clubs (Jim did one for NCCC on 1.85), I can
post on PVRC, but unsure how to reach the international clubs.

**Comment by w7sst** [comment]
I have upgraded the groups.io group to the Premium plan. There was a generous donation of $100 yesterday (Thank you!) and another for $5 today. These reserve dollars paid the first month's billing. We started with the $20/mo. plan. When we get $220 in serve, I can switch us over to the annual billing of $220/yr.

I'll work on an announcement to post on groups.io asking for sponsorship donations to help cover the groups.io hosting fees. I will also update both github and groups.io home pages announcing that the 100-person limit has been removed.

Mike

**Comment by w7sst** [comment]
I added the following to the github home page:

### Notice - 100-person limit on groups.io has been ~resolved~ removed
We have recently removed the 100-person limit from the [MRCE user's group](https://groups.io/g/MorseRunnerCE) on groups.io. If you were unable to subscribe in the past, please visit our user group page and try again. Please click [here](https://groups.io/g/MorseRunnerCE/sponsor) to help cover the MRCE User Group hosting fees; any donation would be appreciated.

**Comment by w7sst** [comment]
Update... One day has passed since we removed the 100-person limit on groups.io.
- 12 donations totaling $230. We have $210 in the reserve account.
- 5 new subscribers
